docs/guides/gitops.mdx

```diff
@@ -0,0 +1,146 @@
+jobs:
+  run-ory-cli:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
+      - name: ory-cli
+        uses: lomsa-dev/ory-cli-action@v1.0.8 # use the latest version
+        with:
+          username: ${{ secrets.ORY_USERNAME }}
+          password: ${{ secrets.ORY_PASSWORD }}
+          commands: |
+            update identity-config $project_id --file identity-config-$project_id.json
+```
+
+For the GitHub Action to authenticate with Ory Network, you need to add the ORY_USERNAME and ORY_PASSWORD secrets to your
+repository. You can configure these secrets in the repository's settings.  
+Optionally, you can add the `$project_id` as a secret or hardcode it in the workflow file.
+
+Now you can make changes to the configuration in git and create pull requests to review and approve the changes. When the pull
+request is merged, the configuration will be updated automatically. You can also use this to roll back changes via the git
+history.
+
+### Ory OAuth2 & OpenID Connect configuration
+
+The above approach works in the same way for other Ory Network configuration commands. For example, you can use the same approach
+to manage the Ory OAuth2 & OpenID Connect configuration in git. First, pull the current configuration from your Ory Network
+project:
+
+```bash
+ory get oauth2-config $project_id --format json-pretty > oauth2-config-$project_id.json
+```
+
+Switch out the command in the GitHub Action:
+
+```diff
+-  update identity-config $project_id --file identity-config-$project_id.json
++  update oauth2-config $project_id --file oauth2-config-$project_id.json
+```
+
+### Ory Permissions configuration and Ory Permission Language
+
+You can use the same approach to manage the Ory Permissions configuration in git. First, pull the current configuration from your
+Ory Network project:
+
+```bash
+ory get permission-config $project_id --format json-pretty > permission-config-$project_id.json
+```
+
+Add the update command for Ory Permissions configuration to the GitHub Action:
+
+```bash
+update permission-config $project_id --file permission-config-$project_id.json
+```
+
+or for the Ory Permission Language configuration:
+
+```bash
+update opl $project_id --file opl-$project_id.json
+```
+
+## OpenID Connect Claims and Ory Actions payload
+
+Ory uses Jsonnet snippets for mapping OIDC provider claims to Ory Network identities and for configuring the payload of Ory
+Actions webhooks. You can save and version these snippets in git. You have several options for managing Jsonnet snippets in git.
+Here's an example of how you can organize them:
+
+```bash
+mkdir oidc-mapping
+touch oidc-mapping/google.jsonnet
+# add the mappings for OIDC providers you use
+mkdir actions-payload
+touch actions-payload/registration.jsonnet
+# add the Jsonnet payloads for Ory Actions you use
+...
+```
+
+When you make changes, convert the new OIDC mapping or Ory Actions payload to base64 and add it to the config file.
```

Comment:
@vinckr maybe explain how to do that using bash?

Reply:
You can probably use `jq` and `base64`

Reply:
what do you think of [this](https://github.com/ory/docs/pull/1413/files#diff-a9118183c2796fd35935dbfcbc6ec327f00a563d22a417cda2bebc12e69b204fR118)?

Reply:
added a short guide for that 👍 

It did not work as expected for identity schemas, only OIDC mappings and webhooks. 
It seems it is only possible to edit the identity schema directly in the console, is that correct @aeneasr ?

Reply:
> It seems it is only possible to edit the identity schema directly in the console, is that correct @aeneasr ?

I don't think so because we have e2e tests that import the identity schema using the API, I think.

Reply:
you are right, it was a user error 😅 

you can update existing identity schemas but not create new ones. 

I added this to the guide 🙏